server/src/Irony.cpp

```diff
@@ -286,6 +286,50 @@ void Irony::complete(const std::string &file,
   std::cout << "(success . t)\n";
 }
 
+namespace {
+
+static bool hasUppercase(const std::string &prefix)
+{
+  for (auto it = prefix.begin(); it != prefix.end(); ++it) {
```

Comment:
Is it std::find_if(b, e, std::isupper) ?

Something like:

    return std::find_if(prefix.begin(), prefix.end(), std::isupper) != prefix.end();

Not sure it compiles, just a suggestion.